content/media/dpg.md

```diff
@@ -0,0 +1,8 @@
+
+![Tazama+DPG Logo](https://www.linuxfoundation.org/hs-fs/hubfs/Tazama%20image.jpg?width=557&height=166&name=Tazama%20image.jpg)
```

Comment:
Did the image render for you? 
<img width="725" alt="Screenshot 2024-10-04 at 6 26 23 PM" src="https://github.com/user-attachments/assets/4e7027d5-7e61-4f5f-a8ab-54bd6f8ad0de">
The URL has query parameters and those sometimes interfere with image rendering in markdown. We might want to host it locally. Although then we loose hs clicks

Reply:
oh, and I think the website is set up for png.

Reply:
well it rendered happily in the markup checker version at least.
I have a version of the logo in the PR, (albiet jpg), in the static/image folder, but I wasn't completely sure of the actual path to stick into the md file so for the time being I had just added the link to the LF web location.